app/package.json

```diff
@@ -3,13 +3,13 @@
   "productName": "GitHub Desktop",
   "bundleID": "com.github.GitHubClient",
   "companyName": "GitHub, Inc.",
-  "version": "0.6.2",
+  "version": "0.6.3-beta6",
   "main": "./main.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/desktop/desktop.git"
   },
-  "description": "Extend your GitHub workflow beyond your browser with GitHub Desktop",
+  "description": "Simple collaboration from your desktop",
```

Comment:
This is the same byline we're using elsewhere and better fits our goals as discussed at the mini summit.